src/commands/jobs/items.ts

```diff
@@ -87,9 +96,16 @@ export = {
 					'status',
 					'processing',
 					'transferred_on'
-				]
-			});
-			output.writeTable(items, outputFormat);
+					]
```

Comment:
This bracket needs to be pulled in 1 tab.

Reply:
will do!